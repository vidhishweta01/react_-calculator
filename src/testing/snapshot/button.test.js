import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../../component/button';

describe('Button component', () => {
  it('matches the snapshot', () => {
    const handleClick = () => 'just for testing';
    const button = renderer.create(<Button handleClick={handleClick} label="1" />).toJSON();
    expect(button).toMatchSnapshot();
  });
});
