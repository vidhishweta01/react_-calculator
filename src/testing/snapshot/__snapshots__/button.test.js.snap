// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component matches the snapshot 1`] = `
<button
  className="button orange normal"
  onClick={[Function]}
  type="button"
>
  1
</button>
`;
